pkg/session_recording/recordings.jsx

```diff
@@ -450,6 +450,42 @@
     let RecordingList = class extends React.Component {
         constructor(props) {
             super(props);
+            this.sortBy = this.sortBy.bind(this);
+            this.sortList = this.sortList.bind(this);
+            this.drawSortArrow = this.drawSortArrow.bind(this);
+            this.state = {
+                sorting_field: "start",
+                sorting_asc: false,
+            };
+        }
+
+        drawSortArrow() {
+            $('#sort_arrow').remove();
+            let type = this.state.sorting_asc ? "asc" : "desc";
+            let arrow = '<i id="sort_arrow" class="fa fa-sort-' + type + '" aria-hidden="true"></i>';
+            $(this.refs[this.state.sorting_field]).append(arrow);
+        }
+
+        sortBy(event) {
+            this.setState({
+                sorting_field: event.currentTarget.id,
+                sorting_asc: !this.state.sorting_asc
+            });
+        }
+
+        sortList(list) {
+            let field = this.state.sorting_field;
+            let sort_way = this.state.sorting_asc;
```

Comment:
The function deals with sorting exclusively, so we don't need to have `sort_` in the variable name, but we need to change from `way` to `asc`. I.e. I think it would be better to call this variable `asc` instead.